lib/db.js

```diff
@@ -9,9 +9,14 @@ module.exports = async () => {
   await client.connect();
   const db = client.db('pa-wildflower-selector');
   const plants = db.collection('plants');
-  await plants.createIndex({
-    "$**": "text"
-  });
+  try {
```

Comment:
This try/catch is a temporary thing to avoid crashes when also testing the PR that changes the text index. Will remove after that one is merged.